package.json

```diff
@@ -104,7 +104,7 @@
     "react-blockies": "^1.4.0",
     "react-display-name": "^0.2.3",
     "react-onclickout": "^2.0.8",
-    "react-spring": "7.2.10",
+    "react-spring": "^8.0.27",
```

Comment:
can we lock this version to 8.0.27 and remove the `^` earlier? Because normally when we are relying on external packages as deps, it is generally suggested to lock versions of the package.

P.S: I would rather do the same thing for react version as well.

Reply:
We could do what Anup suggested, it makes sense, to have all using the same versions.

Reply:
I would rather not do it on gardens-ui as of now since I am working on a PR for updating the react dependencies on gardens-ui.

Reply:
Yes! I almost sure i removed it to put the 7.2.10 version, on that i not fixed because i dont checked if there version its the last one without big break changes. But we can definity do that

Reply:
When we merge this @0xGabi needs to create a new version.